operations/mimir/rollout-operator.libsonnet

```diff
@@ -166,6 +190,80 @@
       namespace: $._config.namespace,
     }),
 
+  zpdb_validation_webhook: if !zpdb_enabled then null else
```

Comment:
I think we should always enable this webhook provided the rollout-operator supports ZPDBs - otherwise, there's a race between creating this webhook and creating the first ZPDB, which might allow us to create an invalid ZPDB.